cmd/generate/config/rules/rule.go

```diff
@@ -68,12 +68,12 @@ func validate(r config.Rule, truePositives []string, falsePositives []string) *c
 	})
 	for _, tp := range truePositives {
 		if len(d.DetectString(tp)) != 1 {
-			log.Fatal().Msgf("Failed to validate (tp) %s %s", r.RuleID, tp)
+			log.Fatal().Msgf("Failed to validate. For rule ID [%s], true positive [%s] was not detected by regexp [%s]", r.RuleID, tp, r.Regex)
```

Comment:
NOTE: Made this message more verbose with brackets around the variables. This helped make the message more useful for me while iterating, but if you don't like this for any reason, happy to revert.